tests/roots/test-ext-autodoc/target/pep695.py

```diff
@@ -0,0 +1,25 @@
+from typing import NewType
+
+
+class Foo:
+    """This is class Foo."""
+
+    ...
+
+type Pep695Alias = Foo
+"""This is PEP695 type alias."""
+
+type Pep695AliasC = dict[str, Foo]  #: This is PEP695 complex type alias with doc comment.
```

Comment:
Change the ruff configuration so that this test file requires python 3.12 as the target version